custom/icds_reports/static/js/icds_app.js

```diff
@@ -122,181 +124,194 @@ window.angular.module('icdsApp', ['ngRoute', 'ui.select', 'ngSanitize', 'datamap
 
         $routeProvider
             .when("/", {
-                redirectTo : '/program_summary/maternal_child',
+                redirectTo: '/program_summary/maternal_child',
             }).when("/program_summary/:step", {
-                template : "<system-usage></system-usage>",
+                template: "<system-usage></system-usage>",
             }).when("/awc_opened", {
-                redirectTo : "/awc_opened/map",
+                redirectTo: "/awc_opened/map",
             })
             .when("/awc_opened/:step", {
-                template : "<awc-opened-yesterday></awc-opened-yesterday>",
+                template: "<awc-opened-yesterday></awc-opened-yesterday>",
             })
             .when("/active_awws", {
-                template : "active_awws",
+                template: "active_awws",
             })
             .when("/submitted_yesterday", {
-                template : "submitted_yesterday",
+                template: "submitted_yesterday",
             })
             .when("/submitted", {
-                template : "submitted",
+                template: "submitted",
             })
             .when("/system_usage_tabular", {
-                template : "system_usage_tabular",
+                template: "system_usage_tabular",
             })
-            .when("/underweight_children", {
-                redirectTo : "/underweight_children/map",
+            .when("/comp_feeding", {
```

Comment:
Are these changes related to the GA work?

Reply:
@calellowitz yes these changes are related. This looks strange because I sorted the urls, but I added only group for the reports urls. For ex: we had ```/underweight_children``` but after my change we will have ```/maternal_and_child/underweight_children```